e2e/packaging_test.go

```diff
@@ -47,7 +47,7 @@ var _ = Describe("E2E TEST:"+checks.CheckPackaging, func() {
 				Score:         checker.InconclusiveResultScore,
 				NumberOfWarn:  1,
 				NumberOfInfo:  0,
-				NumberOfDebug: 4,
+				NumberOfDebug: 0,
```

Comment:
@spencerschrock @laurentsimon Please give feedback about this.

Reply:
I don't think we still have a good way for debug statements with probes

Reply:
Okay. Closing.